statsmodels/stats/tests/test_multi.py

```diff
@@ -355,6 +355,10 @@ def test_tukeyhsd():
     assert_almost_equal(confint, res[:, 1:3], decimal=2)
     assert_equal(reject, res[:, 3]<0.05)
 
+    # check p-values (divergence of high values is expected)
+    small_pvals_idx = [2, 5, 7, 9]
+    assert_almost_equal(res[small_pvals_idx, 3], myres[8][small_pvals_idx], decimal=4)
```

Comment:
It's better to use assert_allclose because it gives more flexibility
assert_almost_equal is old code from before assert_allclose was added to numpy.testing.

(history: we have been using assert_allclose for new code, but are slow, i.e. no concentrated effort, to change the old asserts.)